src/kolibri-cli.ts

```diff
@@ -106,10 +106,19 @@ export class KolibriCli extends Program {
         }
     }
 
+    resultWrapper(result: any) {
+        try {
+          return process.env.KOLIBRI_CLI_FORCE_RAW === '1' ? JSON.stringify(result) : result;
```

Comment:
Instead of reading directly from the process, we have the Environment class which encapsulates the env variables and parses them. 
Then we can use them here instread.